apmpackage/apm/data_stream/app_logs/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -9,3 +9,8 @@ processors:
       name: observer_ids
   - pipeline:
       name: ecs_version
+  - remove:
```

Comment:
Even better, define a pipeline like the one in https://github.com/elastic/apm-server/blob/main/apmpackage/cmd/genpackage/pipelines.go#L155 so that it can be reused in multiple ingest pipelines.